src/components/common/Buttons.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from "@chakra-ui/react";
+
+export const SubmitButton = ({value}:{value: {variant: string; text: string; colorScheme: string;}}) => {
+    return (
+        <Button colorScheme={value.colorScheme} variant={value.variant} type="submit">{value.text}</Button>
+    )
+}
+
+export const ConfirmButton = ({value}:{value: {variant: string; text: string; colorScheme: string;}}) => {
```

Comment:
I think that you can create a type definition (interface) for button data in this file and use in consequent buttons :)